pandas/core/dtypes/dtypes.py

```diff
@@ -670,14 +670,17 @@ class DatetimeTZDtype(PandasExtensionDtype):
 
     type: type[Timestamp] = Timestamp
     kind: str_type = "M"
-    str = "|M8[ns]"
     num = 101
-    base = np.dtype("M8[ns]")
+    base = np.dtype("M8[ns]")  # TODO: depend on reso?
```

Comment:
If base is analogous to "default" I think `M8[ns]` is a reasonable default.

Reply:
agreed but not clear thats what 'base' means.  would be good to have that documented somewhere.

The alternative (that ive put together for (Timestamp|Timedelta).(min|max|resolution) is a property-like that assumes nanoseconds when accessed on the class and is instance-specific when accessed on an instance